HLTrigger/special/src/HLTEcalPhiSymFilter.cc

```diff
@@ -43,11 +49,19 @@ HLTEcalPhiSymFilter::fillDescriptions(edm::ConfigurationDescriptions& descriptio
   desc.add<edm::InputTag>("endcapHitCollection",edm::InputTag("ecalRecHit","EcalRecHitsEE"));
   desc.add<unsigned int>("statusThreshold",3);
   desc.add<bool>("useRecoFlag",false);
-  desc.add<double>("ampCut_barrel",8.);
-  desc.add<double>("ampCut_endcap",12.);
+  desc.add<std::vector<double> >("ampCut_barrelP",{6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,
```

Comment:
Leave the above two in, but mark them as deprecated, like in this example:
```
desc.addOptional<bool>("verbose", false)->setComment("Deprecated and to be removed");
```
Thia bove is for the 74X PR ONLY!!

fillDescriptions of the newly added parameters must be set so as to reproduce the old behaviour in both the 74X and 75X PR.